src/reference/user-surveys.md

```diff
@@ -0,0 +1,36 @@
+---
+title: User surveys
+description: Why users see a survey announcement, how the data is used, and how to disable.
+---
+
+## Why do I see a survey announcement?
+If you have not opted-out of Flutter’s 
+[analytics and crash reporting]({{site.url}}/reference/crash-reporting), 
+you may receive a survey announcement in your IDE. 
+
+We run two types of surveys:
+1. Quarterly surveys - For these surveys, we notify all active
+ Flutter and Dart users to optionally take the survey.
+2. Ad-hoc surveys (experimental): We design and deploy these 
+surveys when we want to learn more about specific topics. 
+You may see a survey announcement if you meet the criteria of 
+the survey  based on data stored in the 
+`.dart-tool/dart-flutter-telemetry.log` file under the home directory
+`~/` (macOS/Linux) or the `<username>\AppData` (Windows) directory. 
```

Comment:
**issue:** Use parallel structure, remove the parenthetical, and tighten the language. Remove location of the file. That's extra info that doesn't aid in understanding here.

```suggestion
1. **Each quarter.** We give all active Flutter and Dart users
  the option to take this survey.
2. **Ad-hoc.** We design and deploy experimental surveys
  when we want to learn more about specific topics. 
  If your telemetry data meets the survey criteria,
  you might see a survey announcement. 
```